.github/workflows/dependabot_automerge.yml

```diff
@@ -0,0 +1,31 @@
+name: "Dependabot Automerge"
+
+on:
+  workflow_run:
+    workflows: ["CI Build"]
+    types:
+      - completed
+
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+
+    if: ${{ github.event.workflow_run.conclusion == 'success' && github.actor == 'dependabot[bot]' }}
+    steps:
+      - uses: actions/github-script@v3
+        env:
+          pull_request_number: ${{ toJSON(github.event.workflow_run.pull_requests[0].number) }}
+        with:
+          script: |
+            github.pullRequests.createReview({
```

Comment:
Should this be `pulls` instead of `pullRequests`? https://octokit.github.io/rest.js/v18#pulls